analyses/cell-type-wilms-tumor-06/notebook-template/01-clustering.Rmd

```diff
@@ -0,0 +1,768 @@
+---
+title: "Evaluation of clustering parameters"
```

Comment:
This notebook isn't exactly evaluating _parameters_, since you fixed those from the beginning. Instead, how about this?

```suggestion
title: "Exploration of clustering results"
```